src/pages/OnboardingEmployees/BaseOnboardingEmployees.tsx

```diff
@@ -30,15 +29,17 @@ function BaseOnboardingEmployees({shouldUseNativeStyles, route}: BaseOnboardingE
     const [selectedCompanySize, setSelectedCompanySize] = useState<OnboardingCompanySizeType | null | undefined>(null);
     const [error, setError] = useState('');
 
-    const companySizeOptions: ListItem[] = useMemo(() => {
```

Comment:
Why did you remove the type? Can we add it back?

Reply:
Added again.